File: L2/C/sala/2.c
#include<stdio.h>
#include<stdlib.h>
#include<string.h>
//
int main(){
	float v[2],n;
	char*inp=(char*)malloc(10*sizeof(*inp)),*split;
	printf("\nInsira as notas do aluno(uma linha): ");
	fgets(inp,10,stdin);
	for(int i=0;split=strtok_r(inp," ",&inp);++i){
		v[i]=atof(split);
	}
	free(inp);
	n=(v[1]+v[0])/2.0;
	if(n<6.0){
		char*sit=(char*)malloc(10*sizeof(*sit)),*inp=(char*)malloc(3*sizeof(*inp));
		sit="Aprovado!";
		printf("Isso nao e bom, faca o exame(fez o exame).\nAgora insira a nota do exame: ");
		fgets(inp,10,stdin);
		n=(v[0]+v[1]+atof(inp))/3.0;
		n<5.0?sit="Reprovado!":sit;
		printf("%s",sit);
	} else {
		printf("Parabens!Te vejo no proximo bimestre!\n");
	}
	printf("Independe da sua condicao, voce tirou %.2f\n\n",n);
	free(inp);
	return 0;
}

File: 1_year/L2/PY/sala/2.py
print()
x=list(map(float,input('Insira as notas: ').split()))
if (x[0]+x[1])/2>=6:
    print('Aprovado! A sua nota foi: {:.2f}'.format((x[0]+x[1])/2))
else:
    x.append(float(input('Insira a nota do exame: ')))
    print('Aprovado por exame') if (x[0]+x[1]+x[2])/3>=5 else print('Reprovado de qualquer jeito')
    print('Ainda assim, a sua nota foi {:.2f}'.format((x[0]+x[1]+x[2])/3))
print()